teacher.rb

```diff
@@ -3,7 +3,7 @@
 class Teacher < Person
   attr_reader :specialization
 
-  def initialize(age, specialization, name = 'UNKNOWN')
+  def initialize(age, specialization, name = 'UNKNOWN', parent_permission = true)
     super(age, name, parent_permission)
     @specialization = specialization
   end
```

Comment:
- [ ] There is a problem listing the `Teacher`, the teachers name is being swaped for the `Specialization`, kindly fix this.


Teacher and Student created properly
![image](https://github.com/ometman/oop-school-library/assets/81932478/6fdd8217-fe1f-46ea-88d6-e6f49bd2e808)

Teacher's name being listed as `Tech` when user selects option 2
![image](https://github.com/ometman/oop-school-library/assets/81932478/3156de99-0d0b-41c0-9d15-a5909ae9b276)


#### Also note that when listing people their ID's are not being listed making it difficult to list rentals using User ID. Kindly fix this.

Reply:
You're right. Thanks. Fixed 😄